randovania/games/samus_returns/logic_database/Surface - East.txt

```diff
@@ -10,12 +10,13 @@ Extra - asset_id: collision_camera_000
   * Extra - start_point_actor_name: ST_SaveStation_004
   > Door to Surface Stash
       Trivial
-  > Top of Mountain
-      Climb Rooms Vertically (No High Jump)
-  > Dock to Surface Crumble Block Challenge (Top)
-      Climb Rooms Vertically (No High Jump) and Lay Power Bomb
   > Dock to Surface - West
       Baby Metroid
+  > Top Left Midair
+      Any of the following:
+          Space Jump or Use Spider Ball
+          # IBJ
```

Comment:
I'd mention why its Intermediate (IBJ twice up the left side until the clouds at the top etc)